OpenSim/Tests/Wrapping/CMakeLists.txt

```diff
@@ -6,4 +6,5 @@ OpenSimAddTests(
     TESTPROGRAMS ${TEST_PROGS}
     DATAFILES ${TEST_FILES}
     LINKLIBS osimTools
+    INCLUDEDIRS "${CMAKE_SOURCE_DIR}/tropter/external/catch"
```

Comment:
It feels a bit gross to pull in `catch` like this, because it's effectively making the wrapping test suite dependent on a transitive dependency (`catch`) that's within another library that's copied in-tree (`tropter`).

I can understand why you'd do it (other places seem to use the header), but I would caution against using `catch` unless it's a direct dependency of OpenSim - otherwise, the risk is that updating the in-tree tropter has a knock-on effect on code that doesn't actually use tropter.